inputfiles/addedTypes.jsonc

```diff
@@ -1303,7 +1320,8 @@
                         },
                         "value": {
                             "name": "value",
-                            "overrideType": "T"
+                            "overrideType": "T",
```

Comment:
`ReadableStreamBYOBReadDoneResult.value` should be `T | undefined`, as [per spec](https://streams.spec.whatwg.org/#ref-for-byob-reader-read%E2%91%A4):
> If the reader is canceled, the promise will be fulfilled with an object of the form `{ value: undefined, done: true }`. In this case, the backing memory region of view is discarded and not returned to the caller.

See also: [Deno](https://github.com/denoland/deno/blob/v1.39.4/ext/web/lib.deno_web.d.ts#L609-L624) and [web-streams-polyfill](https://github.com/MattiasBuelens/web-streams-polyfill/blob/v3.3.2/src/lib/readable-stream/byob-reader.ts#L27-L38).